cmd/bosun/sched/sched_test.go

```diff
@@ -112,7 +112,7 @@ func testSched(t *testing.T, st *schedTest) (s *Schedule) {
 }
 
 var queryTime = time.Date(2000, 1, 1, 12, 0, 0, 0, time.UTC)
-var window5Min = `"2000/01/01-11:55:00", "2000/01/01-12:00:00"`
+var window5Min = `"9.467277e+08", "9.46728e+08"`
```

Comment:
True. However, the tests were already doing this, except for the first query. At least they are consistently bad. We can make this improvement in another PR.